application/api/threads.go

```diff
@@ -16,20 +20,39 @@ type Thread struct {
 	// swagger:strfmt uuid4
 	// unique: true
 	// example: 63d5b060-1460-4348-bdf0-ad03c105a8d5
-	ID uuid.UUID `json:"uuid"`
+	ID uuid.UUID `json:"id"`
+
+	// CreatedAt - the time this thread was started
+	CreatedAt time.Time `json:"created_at"`
```

Comment:
From what I see, `CreatedAt` isn't used by the UI. Is there another reason to include it here?

Reply:
:+1:

Reply:
The UI doesn't currently use it, but it might at some point right? If it isn't an expensive field to include and there isn't a security/sensitivity issue with it is there any reason not to include it?

Reply:
Just trying to streamline where possible, but no, there's no reason I can think of to remove it.